common/utils.cpp

```diff
@@ -295,21 +295,47 @@ void DBusHandler::setDbusProperty(const DBusMapping& dBusMap,
         auto& bus = getBus();
         auto service =
             getService(dBusMap.objectPath.c_str(), dBusMap.interface.c_str());
-        auto method = bus.new_method_call(
-            service.c_str(), dBusMap.objectPath.c_str(), dbusProperties, "Set");
-        if (dBusMap.objectPath ==
-                "/xyz/openbmc_project/network/hypervisor/eth0/ipv4/addr0" ||
-            dBusMap.objectPath ==
-                "/xyz/openbmc_project/network/hypervisor/eth1/ipv4/addr0")
+        if (service == "xyz.openbmc_project.Inventory.Manager")
         {
-            std::cout << " ,service :" << service.c_str()
-                      << " , interface : " << dBusMap.interface.c_str()
-                      << " , path : " << dBusMap.objectPath.c_str()
-                      << std::endl;
+            ObjectValueTree objectValueTree;
+            InterfaceMap interfaceMap;
+            PropertyMap propertyMap;
+            propertyMap.emplace(dBusMap.propertyName.c_str(),
+                                std::get<0>(variant));
+            std::string objPath = dBusMap.objectPath.c_str();
+            std::string toReplace("/xyz/openbmc_project/inventory/system");
+            size_t pos = objPath.find(toReplace);
+            objPath.replace(pos, toReplace.length(), "/system");
+            interfaceMap.emplace(dBusMap.interface.c_str(), propertyMap);
+            objectValueTree.emplace(std::move(objPath),
+                                    std::move(interfaceMap));
+            auto service1 = getService("/xyz/openbmc_project/inventory",
+                                       "xyz.openbmc_project.Inventory.Manager");
```

Comment:
this does not make sense, you already know the service name(at line number 298) when you entered the if statement, there is no need for us to do another mapper call to get the service name here.